src/model/versioning.test.js

```diff
@@ -127,8 +127,15 @@ describe('Versioning', () => {
       expect(Versioning.descriptionRegex).toBeInstanceOf(RegExp);
     });
 
-    test.each(['v1.1-1-g12345678', 'v0.1-2-g12345678', 'v0.0-500-gA9B6C3D0-dirty'])(
-      'is happy with valid %s',
+    test.each(
+      [
+        'v1.1-1-g12345678',
+        'v0.1-2-g12345678',
+        'v0.0-500-gA9B6C3D0-dirty',
+        'v0.0-500a-gA9B6C3D0-dirty',
+        'v0.1.0a-26-g6817b33',
+        'is happy with valid %s',
+      ],
```

Comment:
Here are some more test cases we should use:

2.0.0 
2.0.0-rc.2 
2.0.0-rc.1 
1.0.0 
1.0.0-beta

These are provided by https://semver.org/